doc/CodeStyle

```diff
@@ -20,8 +20,10 @@
  - an output argument which is a pointer to a user variable has _p suffix
  - value types (e.g struct types, integer types) have _t suffix
  - pointer to structs, which are used as API handles, have _h suffix
- - macro arguments begine with _ (e.g _value) to avoid confusion with variables
+ - macro arguments begin with _ (e.g _value) to avoid confusion with variables
  - no leading underscores in function names
+ - header file names must use _fwd.h suffix if contains forward declarations
```

Comment:
@yosefe in case of your last comment - should I change suffix name here from `_fwd.h` to `_def.h`?